decentraland/explorer-desktop
# Feat: show aspect ratio in resolution dropdown

## What does this PR change?

Adds the aspect ratio in the list of resolutions
Removes the duplicated resolutions with different refresh rates and always picks the highest (decided with @olavra  to integrate it like that after the initial change)

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/index.html?renderer=urn:decentraland:off-chain:renderer-artifacts:{branch_name}&{desired_url_params}
2. Go to graphical settings
3. Change the video resolution
4. Verify it is saved and correctly changed

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
